app/assets/scss/_saved-searches.scss

```diff
@@ -0,0 +1,19 @@
+.direct-award-saved-searches {
+
+    .summary-heading {
+        font-size: 22px;
+        margin-bottom: 10px;
+    }
+
+    table {
+        .summary-last-narrow {
+            text-align: left;
+            width: 25%;
+        }
+        
+        &.summary-item-body {
+            margin-bottom: 60px;
+        }
+    }
+
+}
```

Comment:
We apparently like to include an extra blank line at the end of files. Please add one and consider updating your IDE to automatically include it on save.

Reply:
Not a blank line necessarily, but a line terminator on the final line, yes.  It's generally considered good practice, for several (mainly historical) reasons... some discussion here: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Yep, summary table looks like the thing here, and if it needs a variation from what's in the toolkit it's better to make the adjustment there rather than at a per-app level. 
 This is especially important for us now we're splitting out more and more apps - we need to rely on the toolkit to provide a consistent look and feel across all apps. 
 http://alphagov.github.io/digitalmarketplace-frontend-toolkit/summary-table.html

Reply:
At this point adding any new per-app styles for stuff should set off alarm bells I think.